Allow to override keyboard layout and fix zoom in/out tests in GraphEditor (#203)
hexatomic/hexatomic: docs/dev/src/development/automated-tests/ui-integration-tests.md

> # Use interface integration tests

Reviewer: ```suggestion
# Using interface integration tests
```

dependencies of `org.corpus_tools.hexatomic.it.tests`.
Adding it to the feature or product is not enough.

## Issues with keyboard layout and integration tests

Note that the SWTBot-based integration tests try to detect the keyboard layout automatically using the system locale.

> For example, `de_DE.UTF` would result in the keyboard layout `DE_DE`.

Reviewer: ```suggestion
For example, if the computer running the tests has the locale `de_DE.UTF`, SWTBot would detect the keyboard layout `DE_DE`.
```

Make example clearer with regards to the link between locale and keyboard layout.

This can lead to test errors when the locale is not consistent with the keyboard layout (e.g. some programmers prefer English keyboard layouts, which would be `EN_US` or `MAC_EN_US`, while still using their native locale).

> To enforce a specific keyboard layout for the tests, set the `SWTBOT_KEYBOARD_LAYOUT` environment variable to the layout you want to select.

Reviewer: ```suggestion
To enforce a specific keyboard layout for the tests, set the `SWTBOT_KEYBOARD_LAYOUT` environment variable to the layout you want to use.
```

> This can also be helpful if SWTBot does not provide a keyboard layout for the current locale, e.g. there is **no `EN_GB` layout** but you can still enforce the very similar `EN_US` layout.

Reviewer: ```suggestion
This can also be helpful if SWTBot does not provide a keyboard layout for the current locale, e.g., if there is no `EN_GB` layout you can still enforce the very similar `EN_US` layout.
```

Fix grammar.
